docs/products/clickhouse/concepts/data-integration-overview.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Aiven for ClickHouse® data service integration
+sidebar_label: Data integration
+keywords: [data service integration, data source integration, managed credentials integration, managed databases integration, named collections]
+---
+
+import {ConsoleIcon} from "@site/src/components/ConsoleIcons";
+
+Aiven for ClickHouse® supports different types of integration allowing you to efficiently connect with other services or data sources and access the data to be processed.
+
+There are a few ways of classifying integration types supported in Aiven for ClickHouse:
```

Comment:
Perhaps a bit shorter? 

```suggestion
Aiven for ClickHouse classifies integration types:
```

Reply:
Indeed shorter but doesn't seem to convey my intended message. It seems what you're proposing might imply that this classification is an officially designed way of classifying integrations in Aiven for ClickHouse, and it's not. There are no such official ways. This classification is just our proposal of classifying them for better user understanding and more logical overview of all the possible scenarios.